### ISSUE TITLE
Bug:  Abortcontroller  of globa. AbortController

### ISSUE THREAD

**Issue by rambo-panda** [eslint/eslint#15663]
### Environment

Node version:  v16.11.1
npm version:  8.3.1 
Local ESLint version: v5.16.0
Global ESLint version:  none
Operating System:  macos 12.0.1 

### What parser are you using?

Default (Espree)

### What did you do?

<details>
<summary>Configuration</summary>

```
{
         extends: ["airbnb-base", "prettier"],
         rules : {
            "prefer-destructuring": ["error", {"object": true, "array": false}]
         }
        .....
}
```
</details>

```js
const controller = new AbortController(),
  signal = controller.signal;
```

### What did you expect to happen?

`Abortcontroller` is a global variable. should not prompt [no undef](https://eslint.org/docs/rules/no-undef#no-undef) to report an error

### What actually happened?

As mentioned above
```
  3:24  error  'AbortController' is not defined             no-undef
```

### Participation

- [x] I am willing to submit a pull request for this issue.

**Comment by snitin315** [comment]
Yes, `AbortController` is a browser global. Adding `{"env": {"browser": true}}` in your config or `/* eslint-env browser */` at the top of file will fix it for you.

### [You can see it in the online demo](https://eslint.org/demo#eyJ0ZXh0IjoiLyogZXNsaW50LWVudiBicm93c2VyICovXG4vKiBlc2xpbnQgbm8tdW5kZWY6IFwiZXJyb3JcIiovXG5cbmNvbnN0IGNvbnRyb2xsZXIgPSBuZXcgQWJvcnRDb250cm9sbGVyKCksXG4gIHNpZ25hbCA9IGNvbnRyb2xsZXIuc2lnbmFsOyIsIm9wdGlvbnMiOnsicGFyc2VyT3B0aW9ucyI6eyJlY21hVmVyc2lvbiI6ImxhdGVzdCIsInNvdXJjZVR5cGUiOiJzY3JpcHQiLCJlY21hRmVhdHVyZXMiOnsiZ2xvYmFsUmV0dXJuIjp0cnVlfX0sInJ1bGVzIjp7InF1b3RlcyI6Miwibm8tdW5kZWYiOjJ9LCJlbnYiOnsiYnJvd3NlciI6ZmFsc2V9fX0=)
